core/injector/application_injector.hpp

```diff
@@ -91,308 +92,347 @@ namespace kagome::injector {
   template <class T>
   using uptr = std::unique_ptr<T>;
 
-  template <typename... Ts>
-  auto makeApplicationInjector(const std::string &genesis_path,
-                               const std::string &keystore_path,
-                               const std::string &leveldb_path,
-                               Ts &&... args) {
-    using namespace boost;  // NOLINT;
+  // sr25519 kp getter
+  template <typename Injector>
+  sptr<crypto::SR25519Keypair> get_sr25519_keypair(const Injector &injector) {
```

Comment:
we can just move these lambdas outside of makeApplicationInjector function. Maybe it is even better to move them to detail namespace. 
Therefore we will not need to write lambdas that invoke template functions in bind methods